lib/restaurant.rb

```diff
@@ -0,0 +1,17 @@
+class Restaurant
+
+  attr_reader :menu
+
+  def initialize(menu)
+    @menu = menu
+  end
+
+  def submit_order(customer_name, customer_phone_number, order_dishes)
+    order_dishes.each do |dish, quantity|
```

Comment:
Lint/UnusedBlockArgument: Unused block argument - `quantity`. If it's necessary, use `_` or `_quantity` as an argument name to indicate that it won't be used.